core/src/test/scala/wartremover/warts/Any2StringAddTest.scala

```diff
@@ -6,11 +6,13 @@ import org.scalatest.FunSuite
 import org.wartremover.warts.Any2StringAdd
 
 class Any2StringAddTest extends FunSuite {
```

Comment:
You also need to check that explicitly defined `+` methods on types are still allowed.

This should still work:
```scala
class Foo { def +(s: String): String = "Foo" + s }
(new Foo) + "Bar"
```